### PR TITLE
feat: add checks for Py samples

### PR DESCRIPTION
**Pull request by generalova-kate**
### Details:
 - Add linters to track Python samples in GitHub Actions

### Tickets:
 - 50542

### CODE REVIEW

File: .github/workflows/py_checks.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: IE Python Checks
+# TODO: add for IE API, wheels
```
Comment (SDxKeeper) [inline .github/workflows/py_checks.yml:2]: will it be separate PR?

**Review by SDxKeeper — APPROVED** [review]

File: .github/workflows/py_checks.yml (hunk 1 above)
Comment (generalova-kate) [reply .github/workflows/py_checks.yml:2]: yes, after this
